packages/frontend/web-ui/src/game/hooks/useGetGamesWithWinnerPairV1.ts

```diff
@@ -0,0 +1,81 @@
+import { models as apiModels } from '@cornie-js/api-models';
+import {
+  GetGamesV1Args,
+  SerializableAppError,
+} from '@cornie-js/frontend-api-rtk-query';
+import { SerializedError } from '@reduxjs/toolkit';
+import { SubscriptionOptions } from '@reduxjs/toolkit/query';
+
+import { mapUseQueryHookResultV2 } from '../../common/helpers/mapUseQueryHookResultV2';
+import { cornieApi } from '../../common/http/services/cornieApi';
+import { Either } from '../../common/models/Either';
+import { buildGameWithWinnerUserPairArrayResult } from '../helpers/buildGameWithWinnerUserPairArrayResult';
+import { getGamesV1ErrorMessage } from '../helpers/getGamesV1ErrorMessage';
+import { GameWithWinnerUserPair } from '../models/GameWithWinnerUserPair';
+import { useGetWinnerUserV1ForGames } from './useGetWinnerUserV1ForGames';
+
+type UseQuerySubscriptionOptions = SubscriptionOptions & {
+  skip?: boolean;
+  refetchOnMountOrArgChange?: boolean | number;
+};
+
+export interface UseGetGamesWithWinnerPairV1Result {
+  result: Either<string, GameWithWinnerUserPair[]> | null;
+}
+
+function useGetGamesV1(
+  getGamesV1Args: GetGamesV1Args,
+  subscriptionOptions: UseQuerySubscriptionOptions,
+): {
+  result: Either<string, apiModels.GameArrayV1> | null;
```

Comment:
We must propagate the error. Knowing the HTTP response might be required in some flows.